Randomizer/MMRando.py

```diff
@@ -55,6 +56,8 @@
         GeneratorList.append(WeaponGenerator(file))
     if not ParamExistsInArgs(args, '-p'):
         GeneratorList.append(PaletteGenerator(file, Megaman_Default))
+    if not ParamExistsInArgs(args, '-music'):
```

Comment:
Do we need to have it always randomized? I would put it as a +music option.
It's just an opinion not an actual code comment, more of a ux. I wouldn't appreciate different music out of nowhere 😜

Reply:
Oh, yeah! I think I did - so I didn't have to type for testing haha. That reminds me should I add documentation? It always slips my mind